Formula/pydocstyle.rb

```diff
@@ -0,0 +1,36 @@
+class Pydocstyle < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python docstring style checker"
+  homepage "https://www.pydocstyle.org/"
+  url "https://github.com/PyCQA/pydocstyle/archive/refs/tags/6.1.1.tar.gz"
```

Comment:
Can we use the source tarball available from [PyPI](https://pypi.org/project/pydocstyle/#files)? That would make tracking resources easier (and automatic with `bump-formula-pr`/`update-python-resources`).

Reply:
Done! Ready for another review